ultravox/training/configs/llama_70b.yaml

```diff
@@ -0,0 +1,16 @@
+# Training configuration for Ultravox with Meta-Llama-3.1-70B-Instruct and Whisper-Medium
+# This configuration is used alongside release_config.yaml
+
+# TODO: make sure to increase max_duration in mcloud.yaml to 18 hours instead of 6
```

Comment:
We currently don't have a way of influencing `mcloud.yaml` from the configs. I've thought about adding a wrapper script on top that can also give us more flexibility (e.g. additive names or objects). I don't think the need was great enough at the moment though.

Reply:
yeah, I think we'll want something like this, it's annoying that we can't pass in variables easily. For a future PR tho.